### PR TITLE
KPV-2536 :: A bunch of small issues requested by cinfa

### PR DESCRIPTION
**Pull request by iduetxe**

### CODE REVIEW

**Comment by DavidSanFer555** [comment]
Close icon too close to text
Screenshot from 2022-09-19 15-16-00

File: grails-app/i18n/messages.properties [hunk 1]
```diff
@@ -2838,13 +2838,17 @@ uploader.error.serviceError = Error uploading file
...
+uploader.multiFile.removeFile.modal.confirm=Yes, delete
+uploader.multiFile.removeFile.modal.title=Remove file
+uploader.multiImage.replaceImages.actionButton=Replace images on html
+uploader.multiImage.replaceImages.modal.body=Those are replaced image\:
+uploader.multiImage.replaceImages.modal.table.image=Image name
+uploader.multiImage.replaceImages.modal.table.times=Replaced times
+uploader.multiImage.replaceImages.modal.title=Changed images
+uploader.multiImage.replaceImages.upload=Upload images
+widget.loading=Loading...
+contestApplication.modal.confirmVote.title=Voto en {0}
```
Comment (DavidSanFer555) [inline grails-app/i18n/messages.properties]: Spanish messages on english file?????

**Comment by DavidSanFer555** [comment]
"Comrpueba los resutlados en el ranking o en el listado de abajo" sigue sin corregir

Developer branch with POE translations don't merge to issue branch

**Comment by DavidSanFer555** [comment]
Contest screen, with little tile for applications, don't allow any vote action neither put any error on logs.

**Comment by DavidSanFer555** [comment]
Info pre vote is displayed on application screen when try to vote even you voted before

**Review by DavidSanFer555 — CHANGES_REQUESTED** [review]
Several comments on pullrequest without line reference

File: grails-app/i18n/messages.properties (hunk 1 above)
Comment (iduetxe) [reply grails-app/i18n/messages.properties]: Fiuu. Fixed

**Review by DavidSanFer555 — APPROVED** [review]

**Pull request merged**